src/portals/hope/features/config/constants.ts

```diff
@@ -79,6 +79,28 @@ export enum InsuranceType {
   DanishHouse = 'DANISH_HOUSE',
 }
 
+export enum Currency {
+  SEK = 'SEK',
+  NOK = 'NOK',
+  DKK = 'DKK',
+}
+
+export const InsuranceTypeCurrency: Record<InsuranceType, Currency> = {
```

Comment:
Could also create one for the market and use that
```suggestion
export const MarketCurrency: Record<Market, Currency> = {
```

Reply:
`type Contract` has a `market: String`

Reply:
This one is useful as well though! (don't remove it)